TCIABrowser/TCIABrowser.py

```diff
@@ -1410,54 +1453,64 @@ def testBrowserDownloadAndLoad(self):
     widget.getCollectionValues()
     browserWindow = widget.browserWidget
     collectionsCombobox = browserWindow.findChildren('QComboBox')[0]
-    print 'Number of collections: ',collectionsCombobox.count
-    if collectionsCombobox.count> 0:
-      collectionsCombobox.setCurrentIndex(randint(0,collectionsCombobox.count-1))
+    print
+    'Number of collections: ', collectionsCombobox.count
```

Comment:
@mehrtash can you please confirm you tested after your changes?

Why did you add these newlines?